src/app.js

```diff
@@ -8,16 +8,21 @@ const {
   Tray
 } = require('electron')
 const electronDebug = require('electron-debug')
+const appConfig = require('electron-settings')
 const { autoUpdater } = require('electron-updater')
 const { is } = require('electron-util')
 
 const menu = require('./menu')
 const WindowState = require('./state/window')
 
-electronDebug({
-  enabled: true,
-  showDevTools: false
-})
+// Debug mode should only be enabled for development mode
+//   or when a user chooses developer mode
```

Comment:
The two extra whitespaces after `//` seem odd to me, is that intended?

Reply:
That is something I tend to do when I have a long comment broken onto multiple lines to show that it is part of the first sentence.  Then, you can start another sentence and see the difference.

For example.

```js
// Debug mode should only be enabled for development mode
//   or when a user chooses developer mode
// This is another comment that is not part of the previous sentence
```

Reply:
Oh, alright. Fine for me 👍  Although I would end sentences with a full stop to indicate that. 🙈 But in the end it's personal preference, so I don't mind :)